pkgs/tools/networking/pssh/default.nix

```diff
@@ -20,6 +20,8 @@ pythonPackages.buildPythonApplication rec {
     done
   '';
 
+  propagatedBuildDependencies = [ openssh rsync ];
```

Comment:
that would be called `propagatedBuildInputs` - and given these two are already specified explicitly in `bin/*` after `postPatch`, they should be [automatically added as runtime dependencies](https://nixos.org/nix/manual/#ssec-derivation):

> If the build was successful, Nix scans each output path for references to input paths by looking for the hash parts of the input paths. Since these are potential runtime dependencies, Nix registers them as dependencies of the output paths.